src/screens/Register.tsx

```diff
@@ -0,0 +1,167 @@
+import React, { useState } from "react";
+import { ScrollView, StyleSheet } from "react-native";
+import { Button, Icon, Input } from "react-native-elements";
+import { SafeAreaView } from "react-native-safe-area-context";
+import SvgLogo from "../components/SvgLogo";
+import { useTranslation } from "react-i18next";
+
+export default function Register() {
+    const { t } = useTranslation();
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+    const [password2, setPassword2] = useState("");
+    const [username, setUsername] = useState("");
+
+    const handleRegister = () => {
+        console.log(t("startJourney", { returnObjects: true }));
+        if (password !== password2) {
+            return;
+        }
```

Comment:
No visual clue when the two passwords don't equal is probably just a temporary thing, right?